### PR TITLE
Added Java Project and Started working on classes for game functionality

### PR DESCRIPTION
**Pull request by PlanetaryTennis**
Added a JavaProject folder with a java project given the temp. name Game into the main working tree. Into Game a package called FundementalGameMechanics was added to hold core game functionality. To the FMG package an abstract class Die, was added, die is the parent class to all colored die classes. The Red, Green, Blue, and Black Die classes were also added each contains functionality for being rolled, and the sides in a char array.

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haney-oliver) [inline .project]: Is it necessary to add this eclipse project and classpath files? If not remove them and create a .gitignore to exclude these file types to be included in this pr.

File: JavaProject/Game/.classpath [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haney-oliver) [inline JavaProject/Game/.classpath]: ditto

File: JavaProject/Game/.project [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haney-oliver) [inline JavaProject/Game/.project]: ditto

File: JavaProject/Game/bin/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/fundementalgamemechanics/
```
Comment (haney-oliver) [inline JavaProject/Game/bin/.gitignore]: Oh I just saw this git ignore. Just update it to exclude eclipse project files.

**Comment by PlanetaryTennis** [comment]
Closing File as due to git issues. reposting later pull request.

**Pull request closed without merging**